templates/external-dns.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: external-dns
+  labels:
+    kubeaddons.mesosphere.io/name: external-dns
+  annotations:
```

Comment:
We don't have CI set up for this yet: Annotations need to include `catalog.kubeaddons.mesosphere.io/addon-revision` in order for this repository to be supported by the Kubeaddons Catalog. More details in the [README.md](https://github.com/mesosphere/kubeaddons/tree/master/tools/catalog/README.md).

Reply:
@shaneutt Thanks for pointing this out. I have pushed another commit to include `catalog.kubeaddons.mesosphere.io/addon-revision`. PTAL

Reply:
:+1: